templates/cli/lib/commands/push.ts

```
@@ -0,0 +1,2422 @@
+  if (successfullyPushed === 0) {
+    error("No tables were pushed.");
+  } else {
+    success(`Successfully pushed ${successfullyPushed} tables.`);
+  }
+
+  if (cliConfig.verbose) {
+    errors.forEach((e) => console.error(e));
+  }
+};
+
+const pushCollection = async ({}: PushTableOptions = {}): Promise<void> => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)